src/ua/com/javarush/alexbezruk/island/logic/Simulation.java

```diff
@@ -0,0 +1,290 @@
+package ua.com.javarush.alexbezruk.island.logic;
+
+import ua.com.javarush.alexbezruk.island.statistics.Statistics;
+import ua.com.javarush.alexbezruk.island.terrain.Island;
+import ua.com.javarush.alexbezruk.island.terrain.Location;
+import ua.com.javarush.alexbezruk.island.wildlife.WildLife;
+import ua.com.javarush.alexbezruk.island.wildlife.animal.Animal;
+import ua.com.javarush.alexbezruk.island.wildlife.plant.Plant;
+
+import java.io.*;
+import java.lang.reflect.InvocationTargetException;
+import java.util.*;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import java.util.concurrent.TimeUnit;
+
+public class Simulation {
+    private static final int PARA = 2;
+    private static final int PROBABILITY_OF_BIRTH_OF_ANIMAL = 25;
+    private static final int MAXIMUM_AMOUNT_OF_GRASS_EATEN_PER_DAY = 3;
+    private static int day = 0;
+    private static final String PATH_TO_DATA = "resources/data.properties";
+    private static final Properties initialData;
+
+    private final Island island;
+    private final Menu menu;
+
+    static {
+        initialData = getProperties();
+    }
+
+    public Simulation() {
+        island = new Island();
+        menu = new Menu(island);
+    }
+
+    public static Properties getInitialData() {
+        return initialData;
+    }
+
+    public static void start() {
+        Simulation simulation = new Simulation();
+        simulation.settlement();
+        System.out.println("Остров создан и заселен животными");
+
+        Statistics initialStatistics = new Statistics(simulation.island);
+        initialStatistics.outputGeneral();
+        initialStatistics.saveInitialStateOfIsland();
+
+        System.out.println("\nЗапускаем симуляцию жизни острова...\n");
+
+        while (true) {
+            simulation.oneDay();
+            day++;
+            System.out.printf("Прошел день №%d", day);
+            simulation.menu.output();
+        }
+    }
+
+    private static Properties getProperties() {
+        File file = new File(PATH_TO_DATA);
+        Properties properties = new Properties();
+        try (FileReader reader = new FileReader(file)) {
+            properties.load(reader);
+        } catch (IOException e) {
+            System.err.println("Ошибка при чтении данных из properties " + e.getMessage());
+        }
+        return properties;
+    }
+
+    private void settlement() {
```

Comment:
узнал с помощью переводчика только это слово) я бы назвал как-то так)

```suggestion
    private void fillIsland() {
```